contracts/common/mixin/RootChainValidator.sol

```diff
@@ -1,95 +1,78 @@
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.2;
```

Comment:
Looks like this file can be made into a lib.

Reply:
yup the intention of `ChildChainVerifier` was to reuse the same here.

Reply:
Then let's make it point to that?